tests/scripts/thread-cert/Cert_6_1_02_REEDAttach_ED.py

```diff
@@ -77,10 +82,36 @@ def test(self):
         self.simulator.go(5)
         self.assertEqual(self.nodes[REED].get_state(), 'child')
 
-        self.nodes[ED].start()
-        self.simulator.go(5)
-        self.assertEqual(self.nodes[ED].get_state(), 'child')
+        self.nodes[MED].start()
+        self.simulator.go(config.DEFAULT_CHILD_TIMEOUT + 5)
```

Comment:
-> `5 + config.DEFAULT_CHILD_TIMEOUT`
better to add comment to illustrate the reason for waiting extra `DEFAULT_CHILD_TIMEOUT` for better readability?

and I'd suggest to define a constant (e.g. ATTACH_TIME) in config.py for `5`, maybe in a separate PR afterwards?